canonical-web-and-design/cn.ubuntu.com
# add snapdeal takeover

Done:
Added takeover

QA:
Check copy and links of the homepage

Copy:
https://docs.google.com/document/d/1H2ujXaXlWFqVjISu87rb2bF4gDZb1NAJXXm7k31QAZ8/edit#
